client/devdocs/design/index.jsx

```diff
@@ -101,6 +102,7 @@ export default React.createClass( {
 					<InfoPopover />
 					<InputChrono />
 					<Notices />
+					<NoticesGlobal />
```

Comment:
I was wrong when I left that note: we don't actually have a `component/notices` – I didn't realize that this lives in `client/notices/`. This component should probably live somewhere in `components/*/docs` like all of the other examples.

I would probably move this `components/global-notices/docs/` and reference it as `GlobalNotices` in this file. No destination for this file seems obviously correct though, as it seems a bit strange for `global-notices` to live in `components/global-notices/` because it depends on both another component from `client/components/` and `notices`.